index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SelfDev - Making Life Easy</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="top-bar dark-bg">
+            <ul class="topbar-menu">
+                <li class="menu-link">
+                    <a href="" class="gray">
+                        <span class="material-icons">facebook</span>
+                    </a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">
+                        <span class="md material-icons">twitter</span>
+                    </a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">English</a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">My page</a>
+                </li>
+                <li class="menu-link">
+                    <a href="" class="gray">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <div class="overlay">
+            <nav class="main-menu">
+                <a href="index.html" class="logo">
+                    <img src="./images/selDev-logo.png" alt="logo">
+                </a>
+                <ul class="nav">
+                    <li class="nav-link">
+                        <a href="" class="dark">About</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Services</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Coaches</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Resources</a>
+                    </li>
+                    <li class="nav-link">
+                        <a href="" class="dark">Get In Touch</a>
+                    </li>
+                </ul>
```

Comment:
- Good job designing the second navbar but it still needs some finishing touches 💯 
- [ ] Kindly add the link or button with the red border as found on the original design.
- [ ] Also, you'll notice that the active link is colored. It would be great if you could implement this functionality as well so as to improve your UI. Kindly do this for both pages.
- Your design:
![Annotation 2022-05-26 084812](https://user-images.githubusercontent.com/85113567/170447274-386aed7f-f953-4cdb-87ef-acb73b85752a.jpg)
- Required design:
![Annotation 2022-05-26 084829](https://user-images.githubusercontent.com/85113567/170447308-07d37fe0-0fb2-4b8e-8ee8-b708bcde5777.jpg)